test/e2e/commands/selectTab.js

```diff
@@ -0,0 +1,16 @@
+exports.command = function(selector, callback) {
+    var data = this.globals;
+
```

Comment:
const processorToSelectorMapping = {
  'JavaScript': '#panel-javascript',
  'CSS': '#panel-css'
}


            this
             .waitForElementVisible(processorToSelectorMapping[selector])
             .click('#panel-css')